custom_model_runner/README.md

```diff
@@ -138,6 +138,40 @@ Null value imputation   PASSED
 ```
 In case of check failure more information will be provided.
 
+### Runtime Parameters
+The runtime parameters are parameters that the user creates via the DataRobot WEB UI, under the
+custom model assembly tag. When the associated custom model version is being executed
+(testing/deployment) the runtime parameters are delivered to it for further handling. These
+runtime parameters can be read from the model's custom code (.e.g `cusotm.py`) using the
+`RuntimeParameters` class.
+
+Here is an example:
+```
+from datarobot_drum import RuntimeParameters
+
+def load_model(code_dir):
+    url = RuntimeParameters.get("URL_PARAM_1")
+    aws_credential = RuntimeParameters.get("AWS_CREDENTIAL_PARAM_1")
+    ...
+```
+
+During testing and debug in a local development environment, the user can write down the runtime
+parameters into a file and provide it as an input to the `drum` utility.
+
+Here is an example of such a file, whose name can be anything:
+```
+URL_PARAM_1: http://any-desired-location/
```

Comment:
instead of trying to figure out which kind it is, why not use yaml to differentiate.  also, wasn't this supposed to be changed to launch config instead of runtime paramaters?
```yaml
evn_vars:
  env_vars:
    - ENDPT: https://www.example.com
credentials:
    - MYCRED:
        credType: s3
        awsKey: mah-key
```